roofit/roofit/CMakeLists.txt

```diff
@@ -148,6 +148,52 @@ ROOT_STANDARD_LIBRARY_PACKAGE(RooFit
     MathCore
 )
 
+if (roofit-arch-libs)
+  ROOT_LINKER_LIBRARY(RooFitCompute_SSE4.1
+      src/RooFitComputeLib.cxx
+    TYPE
+      SHARED
+    DEPENDENCIES
+      RooFit
+  )
```

Comment:
This would fit in one line. I'm not sure if that's needed, but since those lines will never be extended (e.g. a second source file is added), one doesn't need them to be nicely editable.
I leave it to you if you think those 27 lines should be compressed to 3.